devices/Button/GpioButton.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System.Device.Gpio;
+
+namespace Iot.Device.Button
+{
+    /// <summary>
+    /// GPIO implementation of Button.
+    /// Inherits from ButtonBase.
+    /// </summary>
+    public class GpioButton : ButtonBase
+    {
+        private const int DEFAULT_BUTTON_PIN = 37;
+
+        private GpioController _gpioController;
+        private int _buttonPin;
+        private bool _pullUp;
+
+        private bool _disposed = false;
+
+        /// <summary>
+        /// Initialization of the button.
+        /// </summary>
+        /// <param name="buttonPin">GPIO pin of the button.</param>
+        /// <param name="pullUp">If the system is pullup (false = pulldown).</param>
+        /// <param name="doublePressMs">Max ticks between button presses to count as doublepress.</param>
+        /// <param name="holdingMs">Min ms a button is pressed to count as holding.</param>
+        public GpioButton(int buttonPin = DEFAULT_BUTTON_PIN, bool pullUp = true, int doublePressMs = DEFAULT_DOUBLE_PRESS_TICKS, int holdingMs = DEFAULT_HOLDING_MS)
```

Comment:
I would not create a default GPIO pin. The reason is simple: you have so many possibilities and not a single board having the same button implemented. So I would really like to have this removed. Also remove the constant.

Reply:
re pin: should be fixed.

Reply:
re timestamp: fixed